docs/security/faq.md

```diff
@@ -0,0 +1,188 @@
+# Security FAQ
+
+## Is this project 100% secure?
+
+No project is 100% secure.
+
+## How much work went into securing this project?
+
+A lot. Fundamental protections like isolation, principle of least privilege etc are followed.
+In addition, packages added were reviewed to various degree, weaknesses reported and patched/taken into account.
+No package was added without research and addressing serious security concerns.
+
+For instance, RTL requires nodes to have unique indexes.
+To prevent duplicates, a very reliable script was written that performs all relevant operations atomically.
+Thus RTL indexes can not be duplicated even in case of power failure at any moment (assuming reliable `mv` and `fsync`).
+
+## Which security techniques does this project use?
+
+* Isolation
+* Principle of least privilege
+* Automation that prevents security-related mistakes
+* Declarative programming with secure defaults (security is always opt-out, never opt-in)
+* Code signing and verification
+* Build in isolated ennvironment
+* Determministic builds
+
+## Which security features are rare or unique to this project?
+
+**Important: the following is based on possibly limited information. If you know of a similar project with a security feature
+mentioned here that's not attributed, please report.**
+
+* This is the only project with a clear policy on forks.
+* To my knowledge this project is the only one that uses `btc-rpc-proxy` to protect `bitcoind` out-of-the-box.
+* This project was the first one to implement any kind of RPC filtering for `bitcoind`, long before it was available in `bitcoind` itself.
+  This was later implemented in `nix-bitcoin`, which is the only other project that does this.
+* This project was the first to implement a special macaroon that doesn't allow BTCPayServer to steal sats from LND.
+  This was later copied by `nix-bitcoin` and it's the only other project that does this.
+  Note that this is meant as a compliment to `nix-bitcoin` - copying important security features is a very good thing!
+* This project implements SSO for several applications.
+  The only other known project that does this is BTCPayServer-docker.
+  The application handling authentication is intentionally much, much simpler than BTCPayServer, thus the attack surface is lower.
+* This project sandboxes cookeis of different apps to avoid them being passed between each-other.
+  There's no other known project that does this.
+* The keys of foreign developers are verified using memory-safe GPG implementation [`sqck`](https://github.com/Kixunil/sqck).
+  No other known project does this.
```

Comment:
well, no other project might use sqck, but maybe they do use pgp verification in general?

Reply:
Sure, they do. This section explains all unique features and `sqck` is unique.

Reply:
I realized what you probably meant. Better now?

Reply:
ACK 37c4cd9